python/servo/build_commands.py

```diff
@@ -231,6 +235,7 @@ def build(self, target=None, release=False, dev=False, jobs=None,
 
         if android:
             target = self.config["android"]["target"]
+            opts += ["-p", "libsimpleservo"]
```

Comment:
Rather than this change, I’d prefer to have libsimpleservo in `default-members` in the workspace’s `Cargo.toml`.

(If that crate doesn’t compile on some platforms use `#[cfg]` so that it does, possibly as `#![cfg]` to disable the whole crate.)

Reply:
Done.